src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -69,7 +69,7 @@ internal partial class InlineTemporaryCodeRefactoringProvider : CodeRefactoringP
                 return;
             }
 
-            if (variableDeclarator.Initializer.Kind() == SyntaxKind.RefExpression)
+            if (variableDeclaration.Type.Kind() == SyntaxKind.RefType)
             {
                 // TODO: inlining byref temps is NYI
```

Comment:
We're not enabling these IDE scenarios for ref returns for now per @VSadov comment on #12838.
Right now the tool tip is visible and IDE crashes when user tries that. My PR is disabling that scenario for the user. I don't think enabling quick fixes for ref returns is planned in the immediate future.